### PR TITLE
Compact lumi pair spectrometer

### PR DESCRIPTION
**Pull request by dhevang**
### Briefly, what does this PR introduce?

Implement a new luminosity dipole magnet based on design from BNL engineer Peng Xu.
The stronger magnet allows the pair spectrometer to be much more compact.  

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

**Review by wdconinc — APPROVED** [review]
This looks like a lot of nice work.

File: compact/far_backward/lumi/lumi_magnets.xml [hunk 1]
```diff
@@ -1,86 +1,95 @@
 <!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
-<!-- Copyright (C) 2022 Justin Chan -->
+<!-- Copyright (C) 2023 Justin Chan -->
```
Comment (veprbl) [inline compact/far_backward/lumi/lumi_magnets.xml:2]:
```suggestion
<!-- Copyright (C) 2022, 2023 Justin Chan, Dhevan Gangadharan -->
```

**Comment by wdconinc** [comment]
> It appears to be related to some change in the beamline magnet volumes.

That's why I asked @ajentsch to double check with you that [his beamline magnet changes](https://github.com/eic/epic/pull/492) wouldn't affect the backward region :-) There were no overlaps [after those changes were merged in main](https://github.com/eic/epic/actions/runs/5803296896/job/15731259751).

**Comment by ajentsch** [comment]
> > It appears to be related to some change in the beamline magnet volumes.
> 
> That's why I asked @ajentsch to double check with you that [his beamline magnet changes](https://github.com/eic/epic/pull/492) wouldn't affect the backward region :-) There were no overlaps [after those changes were merged in main](https://github.com/eic/epic/actions/runs/5803296896/job/15731259751).

How could my changes cause overlaps on the opposite side of ePIC?? (I did communicate the changes to both @dhevang and @simonge)

**Comment by dhevang** [comment]
Yeah, Alex mentioned the change to me.  
The tests that I triggered 18 hours ago went through fine without this overlap error.  The one I triggered this morning had a problem though.  All I did in between was change a copyright comment.

**Comment by wdconinc** [comment]
> How could my changes cause overlaps on the opposite side of ePIC?

Because you changed a magnet geometry description that is also used on the opposite side of ePIC.

 > The tests that I triggered 18 hours ago went through fine without this overlap error.

*Not* getting an overlap error is not a guarantee that there are *no* overlaps (but getting an overlap error is a guarantee that there is an overlap):
- overlap checking is a statistical procedure, not an analytic procedure,
- there is a tolerance, and for overlaps that are close to the tolerance you may 'sneak through' without triggering an overlap error.

**Comment by dhevang** [comment]
Well, the last 2 CI tests that I triggered went through fine (no FF overlap error).  Despite the statistical limitation to these checks, should we merge this into main or wait till a fix comes?  It should affect all other PRs too right?

**Comment by ajentsch** [comment]
I meant to fix it yesterday, but then another issue derailed me yesterday
and today. I should be able to get it fixed tomorrow. Sorry about that.

**Review by wdconinc — APPROVED** [review]

**Comment by dhevang** [comment]
@wdconinc initiated two new tests today and both failed with the same overlap error:
  Overlap is detected for volume AV_11293!VacuumMagnetElement_assembly_51#51!v_b1pf_vacuum_5#5:57 (G4Cons) with AV_11287!Q2PF_BeamlineMagnet_assembly_29#29!yoke_vol_0#0:30 (G4Tubs)
  Concerning the issue, I suppose the problem is that the G4Tubs do not lie on the same straight line, because there's the b1pf dipole.  The vacuum tube inside the b1pf is at a slight angle wrt the Q2PF tube.  So a tiny corner of the vacuum tube edge probably hits the Q2PF.  Although not the most elegant solution, could one simply modify the length or radius of the vacuum by like 0.5 mm?  @ajentsch is that a viable solution?

**Pull request merged**